docs/docs/deploy/environment-variables.mdx

```diff
@@ -0,0 +1,131 @@
+---
+id: environment-variables
+title: Adding Environment Variables
+abstract: |
+    Rasa offers advanced configuration through environment variables.
+    Find the full reference here.
+---
+
+import RasaProLabel from "@theme/RasaProLabel";
+
+To add extra environment variables to your containers, add the
+environment variable to an `extraEnvs` array on the service in your `values.yml`. These
+values can be provided directly or refer to [secrets](../production/secrets-managers.mdx).
+The `extraEnvs` parameter is available for all services.
+For example:
+
+```yaml
+# rasa: Settings common for all Rasa containers
+rasa:
+  # other configured parameters
+  # ...
+  extraEnvs:
+  - name: "LOG_LEVEL"
+    value: "warning"
+```
+
+## Rasa Pro and Rasa Open Source
+
+When running Rasa Pro, the `RASA_PRO_LICENSE` environment variable needs to contain
+the Rasa Pro license key.
+
+### Backing services
+
+| Environment Variable              | Description                                                                                                                                                                                                           | Default           |
+| :-------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------- |
+| `POSTGRESQL_SCHEMA`               | The postgres schema for the [`SQLTrackerStore`](../production/tracker-stores.mdx#sqltrackerstore).                                                                                                                    | `"public"`        |
+| `POSTGRESQL_POOL_SIZE`            | The limit of the number of open connections for the [`SQLTrackerStore`](../production/tracker-stores.mdx#sqltrackerstore).                                                                                            | `50`              |
+| `POSTGRESQL_MAX_OVERFLOW`         | The [maximum overflow](https://docs.sqlalchemy.org/en/13/core/pooling.html#sqlalchemy.pool.QueuePool.params.max_overflow) of pool size for the [`SQLTrackerStore`](../production/tracker-stores.mdx#sqltrackerstore). | `100`             |
+| `RABBITMQ_SSL_CLIENT_CERTIFICATE` | The path to the SSL client certificate for the [`PikaEventBroker`](../production/event-brokers.mdx#pika-event-broker).                                                                                                |                   |
+| `RABBITMQ_SSL_CLIENT_KEY`         | The path to the SSL client key for the [`PikaEventBroker`](../production/event-brokers.mdx#pika-event-broker).                                                                                                        |                   |
```

Comment:
These envars can also be set by endpoints.yaml, we should mention that. I wonder if they were added as a envar because of any security reasons? Maybe easier to keep envars as secrets?

Reply:
I think that should be mentioned in the docs I linked. I'll check if that's already the case.

Reply:
🤔 actually it doesn't seem that `RABBITMQ_SSL_CLIENT_KEY` and `RABBITMQ_SSL_CLIENT_CERTIFICATE` are configurable via `endpoints.yml`. So we're good?

Reply:
oh cool then my bad! I just checked again and these two environment variables are also documented on their respective pages